packages/dialtone-css/lib/build/less/components/notice.less

```
@@ -41,6 +41,51 @@
     box-shadow: var(--notice-box-shadow);
 }
 
+.d-toast-alternate {
+    //  Component CSS Vars
+    //  ------------------------------------------------------------------------
+    --notice-alternate-color-background: var(--dt-color-neutral-white);
+    --notice-alternate-color-text: var(--dt-color-neutral-black);
```

Change to semantic colors to make it work in both light and dark mode.

```suggestion
    --notice-alternate-color-background: var(--dt-color-surface-primary);
    --notice-alternate-color-text: var(--dt-color-foreground-primary);
```